### PR TITLE
Add createUser, profile, loginForm and userForm

### PR DESCRIPTION
**Pull request by NikitaBondarevv**

### CODE REVIEW

File: src/components/header/authorizedUser/styles.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+.dropDown a:hover {
+  text-decoration: underline 1px;
+  text-underline-offset: 5px
+}
+
+.dropDown li {
+  font-size: 26px;
+  line-height: 24px;
```
Comment (iliatcymbal) [inline src/components/header/authorizedUser/styles.css:35]: how comes line height is less then font size?

File: src/pages/products/editableText/styles.css [hunk 2]
```diff
@@ -1,14 +1,14 @@
 .description {
-  font: 25px/30px "MyriadProRegular";
-  min-width: 550px;
-  min-height: 120px;
+  font: 25px "MyriadProRegular";
+  width: 713.48px;
```
Comment (iliatcymbal) [inline src/pages/products/editableText/styles.css]: why do you need `.48` here?

File: src/interfaces/IData.ts [hunk 3]
```diff
@@ -2,4 +2,6 @@ export interface IData {
   categories?: number
   products?: number
   publishedCategories?: number
+  email?: string
+  password?: string
```
Comment (iliatcymbal) [inline src/interfaces/IData.ts]: it is not clear what describes this interface. What is data?

File: src/components/header/authorizedUser/styles.css (hunk 1 above)
Comment (NikitaBondarevv) [reply src/components/header/authorizedUser/styles.css:35]: This is mistake. There should generally be a font-size of 25 pixels

File: src/pages/products/editableText/styles.css (hunk 2 above)
Comment (NikitaBondarevv) [reply src/pages/products/editableText/styles.css]: Tried to adjust to span

File: src/components/header/index.tsx [hunk 4]
```diff
@@ -1,27 +1,51 @@
...
+    <header className={styles.header}>
+      <Link to="/">
+        <img src={logo} className={styles.logo} alt="logo" />
+      </Link>
+      <nav>
+        {
+          isAuthenticated
+            ? <ul className={styles.list}>
+              {links.map((link, index) => (
+                <li key={index}>
```
Comment (iliatcymbal) [inline src/components/header/index.tsx]: there should be padding for nested jsx

File: src/pages/products/editableText/styles.css (hunk 2 above)
Comment (iliatcymbal) [reply src/pages/products/editableText/styles.css]: looks like useless

File: src/pages/products/index.tsx [hunk 5]
```diff
@@ -6,15 +6,13 @@ const descriptionText = `Enjoy new movie-themed thrills ad a favorite Radiator S
...
   <div className={styles.productInformation}>
     <span className={styles.title}>
-      TITLE:
-      <EditableText text="TOW TRACK" stylesInput={styles.editableTitle} />
+      TITLE: <EditableText text="TOW TRACK" stylesInput={styles.editableTitle} />
     </span>
     <span className={styles.price}>
-      $
-      <EditableText text="32" stylesInput={styles.editablePrice} />
+      $ <EditableText text="32" stylesInput={styles.editablePrice} />
```
Comment (iliatcymbal) [inline src/pages/products/index.tsx]: `EditableText` is a component, so it should be in Components

File: src/pages/products/styles.css [hunk 6]
```diff
@@ -13,27 +13,28 @@
...
-  font: 25px/25px "MyriadProRegular";
+  font: 25px/30px "MyriadProRegular";
   background: none;
   width: 130px;
   height: 25px;
   border: 1px solid rgb(202, 202, 202);
   outline: none;
   color: rgb(202, 202, 202);
+  padding: 0 0 0 5px;
 }
```
Comment (iliatcymbal) [inline src/pages/products/styles.css]: why do you keep styles for one component in another component?

Comment (NikitaBondarevv) [reply src/pages/products/styles.css]: Because styles come from props and they are different. The difference in width

Comment (NikitaBondarevv) [reply src/pages/products/styles.css]: i fix this

File: src/interfaces/IData.ts (hunk 3 above)
Comment (NikitaBondarevv) [reply src/interfaces/IData.ts]: This is what the server sends or we send it to the server for verification

**Pull request merged**